app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @like.author = current_user
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new
+    if @like.save
+      flash[:notice] = 'Like added successfully!'
+      redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+    else
+      flash[:alert] = "Couldn't add a like!"
+      puts @like.errors.full_messages
+      render :new, status: :unprocessable_entity
+    end
+  end
```

Comment:
Nice job adding the create function to the likes controller, however, when I tried adding likes, it didn't work. I even checked the rails console to see the likes records that are saved, and it returned an empty list:
![image](https://github.com/jonas-45/blog-app/assets/93520178/fd1f0428-198c-49e4-aced-af932f8db27f)
Kindly fix this issue.

Reply:
okay on it. thanks